### PR TITLE
test: improve web e2e test stability

### PR DESCRIPTION
**Pull request by hamidzr**
## Commentary (optional)

I separated this out from push architecture branch
more details in the linked issue

## Checklist

- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/tests/tests/StepImplementation.ts [hunk 1]
```diff
@@ -211,6 +217,7 @@ export default class StepImplementation {
   @Step('Navigate to experiment list page')
   public async navigateToExperimentList() {
     await goto(`${BASE_URL}/experiments`);
+    await t.waitFor(async () => await t.$(selectors.antdTable).exists());
```
Comment (hamidzr) [inline webui/tests/tests/StepImplementation.ts:220]: this should be simplified

Comment (hamidzr) [reply webui/tests/tests/StepImplementation.ts:220]: I found the same thing in their examples. I'm not sure if it's a bad example or if this necessary. https://docs.taiko.dev/api/waitfor/ 

you'd think `await t.waitFor(t.$(selectors.antdTable));` would be thing

**Comment by hamidzr** [comment]
test failures are all unrelated to webui. I'll rebase to see if there are any fixes in

File: webui/tests/bin/createUserAndExperiments.py [hunk 2]
```diff
@@ -60,6 +60,7 @@ def createExperiment(directory, config_file, count):
         "det",
         "experiment",
         "create",
+        "--paused",
```
Comment (hkang1) [inline webui/tests/bin/createUserAndExperiments.py:63]: 👍

File: webui/tests/tests/StepImplementation.ts (hunk 1 above)
Comment (hkang1) [reply webui/tests/tests/StepImplementation.ts:220]: might we worth testing it out?

**Review by hkang1 — APPROVED** [review]
Code wise it looks good, but I haven't been able to get this up and tested locally due to some weird setup issue. :(

**Review by hkang1 — CHANGES_REQUESTED** [review]
oops approved the wrong PR, sorry!

**Review by hkang1 — APPROVED** [review]
The code looks good but I'm currently not able to test it locally due to weird setup issues :(

**Comment by hkang1** [comment]
> oops approved the wrong PR, sorry!

er it was the right one, nm!

**Review by hkang1 — COMMENTED** [review]
Finally got the local env setup and it ran great the few times I ran it.

File: webui/tests/tests/StepImplementation.ts (hunk 1 above)
Comment (hamidzr) [reply webui/tests/tests/StepImplementation.ts:220]: there are a couple of places we do this I think I tested it but I can't remember I'll retry.

Comment (hamidzr) [reply webui/tests/tests/StepImplementation.ts]: b790a0954988a265ff16dfbd3ea4713803bb2736

**Pull request merged**